### ISSUE TITLE
Implement documentation page

### ISSUE THREAD

**Issue by Kissaki** [mumble-voip/mumble-www#35]
Intro to our different types of configuration

**Comment by Kissaki** [comment]
- [ ] Collect some requirements - mandatory and optional/wishes
- [ ] Add a docs landing page
- [ ] Migrate slice docs
- [ ] Migrate server setup guide
- [ ] Map out migration order (ACL, features, translation, FAQ, scripting, theming, development)
- [ ] Figure out if and how to integrate (or just link) protocol documentation

**Comment by Kissaki** [comment]
# Mandatory Requirements

* Figure out if and how to take over existing content (content history and copyright/author naming), maybe for each page individually

# Wishes

* Edit page link to allow easy submitting of PRs

# General References of good documentation pages

* docs.microsoft.com
** GitHub integration
** Page and in-page navigation and index
** Edit page via PR
** GitHub issues displayed and linked on pages

**Comment by davidebeatrici** [comment]
`docs.microsoft.com` is definitely what we should take inspiration from.

They use markdown for pages, meaning that the end result (without the theme/style) can be previewed.

https://github.com/MicrosoftDocs/cpp-docs

**Comment by Kissaki** [comment]
I think I will make this a multi part and partial process.

Migrate some content, then establish frontend tech like the page navigation, then menu navigation, and only then more content.

Seeing some of the content, I want to also improve the structure. Adding a hierarchical structure to our shitty wiki structure should yield a great improvement IMO.

**Comment by Kissaki** [comment]
Additional references:

* [bep/docuapi](https://github.com/bep/docuapi/) is a Hugo theme for documentation
* [SCSS](https://sass-lang.com/documentation)

**Comment by toby63** [comment]
Once https://github.com/mumble-voip/mumble/pull/4628 is ready, it could be included.

**Comment by Krzmbrzl** [comment]
@vrifox maybe you can use this issue as some sort of context for potential future discussions/projects/involvement/you name it :)

**Comment by Kissaki** [comment]
I created a tool that converts page + revision + author data to git commits and markdown files with frontmatter. One result see #187 

@Krzmbrzl and @vrifox went through the wiki pages to determine which pages would even be potentially worthwhile to migrate - current draft/selection at https://github.com/Krzmbrzl/mumble-wiki-export

**Comment by Krzmbrzl** [comment]
@restitux was the one who took initiative with the wiki export (tagging to include into the conversation here)

**Comment by Kissaki** [comment]
I'll list a table of the pages included in the mumble-wiki-export that are potentially worth to then assess and migrate them:

| wiki page                            | Desc                             | Assessment | Status              |
| ------------------------------------ | -------------------------------- | ---------- | ------------------- |
| 3rd party applications               |                                  |            |                     |
| acl tutorial                         |                                  |            | resolved by PR #188 |
| acl and groups                       |                                  |            | resolved by PR #188 |
| audio                                |                                  |            |                     |
| channel viewer protocol              |                                  |            |                     |
| client settings                      |                                  |            |                     |
| configuring murmur                   |                                  |            |                     |
| faq                                  |                                  |            |                     |
| features                             |                                  |            |                     |
| how to set up an ldap authentication |                                  |            |                     |
| ice                                  |                                  |            |                     |
| installing mumble                    |                                  |            |                     |
| language translation                 |                                  |            |                     |
| link                                 |                                  |            |                     |
| logitech g-keys                      |                                  |            |                     |
| mumble import certificate            |                                  |            |                     |
| mumble url                           |                                  |            |                     |
| mumble acl example                   | how-to force guests into channel |            |                     |
| mumble project                       |                                  |            |                     |
| mumbleguide                          |                                  |            |                     |
| mumla                                |                                  |            |                     |
| murmur.ini                           |                                  |            |                     |
| murmurguide                          |                                  |            |                     |
| running murmur                       |                                  |            |                     |
| SRV record                           |                                  |            |                     |
| skinning                             |                                  |            |                     |
| themes                               |                                  |            |                     |
| third party applications             |                                  |            |                     |
| usage of html and css in mumble      |                                  |            |                     |

**Comment by Kissaki** [comment]
#213 purged the wiki. So the doc migration concerns are no longer relevant/valid.

I'm not quite satisfied with the docs structure and presentation of our website, and most notably a user guide into setting up a server is missing, and the protocol docs link is still open (we previously had an individual ticket #59 for it), but this ticket is too broad and largely outdated.

Closing.
